eng/tox/install_depend_packages.py

```diff
@@ -43,7 +43,7 @@
     "azure-core": "1.11.0",
     "requests": "2.19.0",
     "six": "1.12.0",
-    "cryptography": "3.3.2",
+    "cryptography": "38.0.3",
```

Comment:
Is this correct? That is a pretty large version jump.

Reply:
It looks like they had a versioning scheme change in 2021 that bumped them from 3.4 to 35.0.